packages/perseus/src/widgets/expression.tsx

```diff
@@ -352,10 +352,6 @@ export class Expression extends React.Component<Props, ExpressionState> {
         return Expression.getUserInputFromProps(this.props);
     };
 
-    change: (...args: any) => any | undefined = (...args: any) => {
-        return Changeable.change.apply(this, args);
-    };
-
```

Comment:
Just a quick thought while you're working on this: Expression is exported from the Perseus package. Since some methods are prefixed with `_` and this one isn't, it makes me worried that this might be part of the public API for widgets.

I couldn't find an example of where we use anything like `widget.change` or `expression.change`, but I'm wondering how we can be sure that we're not unintentionally removing a method that's meant to be part of the public API? I see `type Widget = any;` which is a bummer; do we document what's meant to be outward facing?

Reply:
I was worried about this as well, but I checked an all uses of `.change` and I don't see anywhere where it's being passed to something without directly calling it.

I wouldn't expect methods on widgets or editors to be called directly outside of the perseus codebase.

Reply:
I tried setting `export type Widget = {type: string};` to see if `.change()` was ever called on any widget, and it's not.  The only methods we call directly on widget components are `passageWidget.getLineCount()`, `passageWidget.getLineCount()`, `passage.getReference(this.props.referenceNumber)`.